examples/h2o-classification/h2o-example.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import local bentoml repository\n",
+    "import sys\n",
+    "import os\n",
+    "sys.path.insert(0, os.path.join(os.getcwd(), \"..\", \"..\"))\n",
+    "import bentoml\n",
+    "\n",
+    "print(bentoml.__file__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# H2O Classification"
```

Comment:
maybe add a few more markdown cells with some context/comments to this notebook?

Reply:
done